pkgs/development/interpreters/yex-lang/default.nix

```diff
@@ -1,21 +1,25 @@
-{ lib, stdenv, rustPlatform, fetchFromGitHub }:
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+}:
 
 rustPlatform.buildRustPackage rec {
   pname = "yex-lang";
-  version = "unstable-2021-12-25";
+  version = "0.pre+date=2022-05-10";
```

Comment:
what is this version formatting? I've not seen it before

Reply:
Since it's still in discussion should it actually be used in nixpkgs already?
Does this play nicely with repology as it is currently?

Reply:
No serious problems were reported...